mono/profiler/proflog.c

```diff
@@ -885,7 +892,7 @@ gc_reference (MonoObject *obj, MonoClass *klass, uintptr_t size, uintptr_t num,
 	int i;
 	uintptr_t last_offset = 0;
 	//const char *name = mono_class_get_name (klass);
-	LogBuffer *logbuffer = ensure_logbuf (20 + num * 8);
+	LogBuffer *logbuffer = ensure_logbuf (1 + 10 + 10 + 10 + 10 + num * (10 + 10));
```

Comment:
I still like for us to address this. A bucket of magic numbers do us no good. Specially when they seem to be repeated all over the place.

How about using structs with packing/alignment set 1 and have size functions based on sizeof?